packages/smart-contracts/scripts-create2/deploy-request-deployer.ts

```diff
@@ -29,3 +29,20 @@ export async function deployDeployer(hre: HardhatRuntimeEnvironment): Promise<vo
     console.error(e);
   }
 }
+
+// Verifies the RequestDeployer (useful if the verification failed after deployment)
+export async function verifyDeployer(hre: HardhatRuntimeEnvironment): Promise<void> {
+  try {
+    const RequestDeployer = await hre.ethers.getContractAt(
+      'RequestDeployer',
+      '0xE99Ab70a5FAE59551544FA326fA048f7B95A24B2',
```

Comment:
what is this address?

Reply:
It's the address of the `RequestDeployer` contract, same on all networks

Reply:
it should be a variable or should have a comment

Reply:
There is one in the hardhat config file actually. Maybe we should put it elsewhere and export it

Reply:
Good point. Okay I'll fix this.